docs/options/index.rst

```diff
@@ -76,6 +76,71 @@ Configuration file example:
 
   auth_port: 2022
 
+.. _configuration-options-enable-metrics:
+
+``enable-metrics``
+-------------
+
+Enable metrics collection exposition (Prometheus metrics).
+
+Default: ``false``
+
+CLI usgae example:
+
+.. code-block:: bash
+
+  $ napalm-logs --enable-metrics
+
+Configuration file example:
+
+.. code-block:: yaml
+
+  metrics_enabled: true
+
+.. _configuration-options-metrics-address:
+
+``metrics-address``
+-----------
+
+The IP address to use to listen for all incoming metrics requests. This is the
+address that the Prometheus HTTP server exposes metrics from.
+
+Default: ``0.0.0.0``.
+
+CLI usage example:
+
+.. code-block:: bash
+
+  $ napalm-logs --metrics-address 172.17.17.1
+
+Configuration file example:
+
+.. code-block:: yaml
+
+  metrics_address: 172.17.17.1
+
+.. _configuration-options-metrics-port:
+
+``metrics-port``
+-------------
+
+The port to listen on for incoming metrics requests. This is the port that
+the Prometheus HTTP server exposes metrics from.
+
+Default: ``9215``
```

Comment:
`9215` is listed on https://github.com/prometheus/prometheus/wiki/Default-port-allocations as `Prometheus Graphite Bridge`. It would be good to use a unique port number and add it to that this.

Reply:
Good catch. I have reserved 9443.